test/use-checkout-pricing.test.js

```diff
@@ -153,16 +151,6 @@ describe('useCheckoutPricing', function() {
   });
 
   describe('adjustments', () => {
-    it('should call checkoutPricing.adjustment for each adjustment', async () => {
```

Comment:
This test is nearly identical to the one below it and was removed for being redundant.